functions/Test-DbaDatabaseOwner.ps1

```diff
@@ -71,9 +69,9 @@ function Test-DbaDatabaseOwner {
         [Alias("Databases")]
         [object[]]$Database,
         [object[]]$ExcludeDatabase,
-        [string]$TargetLogin = "sa",
+        [string]$TargetLogin ,
         [Switch]$Detailed,
-        [switch]$EnableException
+        [switch][Alias('Silent')]$EnableException
```

Comment:
Can you format this one to match the other parameters, I've been doing this in the commands as I touch them.

```powershell
[Alias('Silent')]
[switch]$EnableException
```

That is the format most tools use but a few folks that are using PSS have it set different.